Models/Polygon.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Models {
     public class Polygon : IReadOnlyList<Position> {
-        private readonly List<Position> _positions;
+        public readonly List<Position> _positions;
```

Comment:
Hoezo is deze public gemaakt?

Reply:
had het vergeten terug te zetten naar private, oops